atlas_densities/app/mtype_densities.py

```diff
@@ -221,19 +221,12 @@ def standardize_probability_map(probability_map: "pd.DataFrame") -> "pd.DataFram
 
     Returns: a data frame complying with all the above constraints.
     """
-    probability_map = probability_map.copy()
-
-    # The rows referring to layer 6b or to the Vip molecular marker are not used by the algorithm
-    # computing the mtype densities.
-    mask = [("Vip" in row_label) or ("6b" in row_label) for row_label in probability_map.index]
-    indices = probability_map.index[mask]
-    probability_map.drop(indices, inplace=True)
 
     def standardize_row_label(row_label: str):
         """
         Lowercase labels and use explicit layer names.
         """
-        splitting = re.split("-|_| ", row_label)[:-2]  # remove unused Creline information
+        splitting = re.split("_", row_label)  # remove unused Creline information
```

Comment:
the tests in `tests/app/test_mtype_densities.py` and  `tests/densities/test_mtype_densities_from_map.py` should have the naming updated to the new convention

It appears it's been regularized to be `L1_Gad2` from `L1 Gad2-IRES-Cre`? Can we make that clear in the commit message?

Reply:
Fixed the tests with my last commit. Forgot to put it in the commit message.